### PR TITLE
Confirm email and resend email 

### PR DESCRIPTION
**Pull request by JorgeVidal29**
Now can confirm and resend confirmation email

**Description**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Resolves #(replace parentheses and the stuff here with the issue number, if there isn't one, create one.) 

**Type of change**
Check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Fix or feature that would cause existing functionality to not work as expected
- [ ] This change requires a update in the design document

**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design document (if applicable)
- [x] My changes generate no new warnings
- [x] Pulled updated master branch (if changed since branch creation) and corrected any conflicts, if any occur.

### CODE REVIEW

File: src/login/ConfirmEmail.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+					console.log(data);
+					this.setState({message: getErrorFromObject(data)});
+				}
+				else {
+					console.log(data);
+					this.setState({message: "Confirmed! Please go to our login page and login"});
+				}
+			})
+		).catch(
+			(error) => {
```
Comment (SecondThread) [inline src/login/ConfirmEmail.js:56]: Don't need the parens around `error`

File: src/login/ConfirmEmail.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+//@flow strict
+
+import * as React from 'react';
+import getErrorFromObject from '../globalComponents/getErrorFromObject.js';
+
+type Props = {
+	match: {
+		params: {
```
Comment (SecondThread) [inline src/login/ConfirmEmail.js:8]: Oh that's really cool, I didn't know you could do that

**Review by SecondThread — APPROVED** [review]
I didn't test it, so that should happen by someone, but the code looks quite good

**Review by Derplime — APPROVED** [review]
Added deleting the token from the DB upon successful confirmation to the PR, also updated some response prompts on the frontend. See [here](https://github.com/RickLeinecker/Fortuna/pull/159/commits/ee3edeb68cde03e631380f6c13bb716462813fb0)

Other than that, just look over David's recommendations and you're good.

**Pull request merged**